shuup/core/models/_labels.py

```diff
@@ -32,4 +32,5 @@ class Meta:
         verbose_name_plural = _('labels')
 
     def __str__(self):
-        return self.safe_translation_getter("name", default=self.identifier)
+        from django.utils.encoding import force_text
```

Comment:
Any reason why the force text is imported under __str__?

Reply:
just to not add another import in the header which increases the time to load the progress. but yeah, that import is already added somewhere else no so big deal, I can change this